draft-ietf-modpod-group-processes.md

```diff
@@ -247,32 +247,20 @@ should be approached.
 
 ## IETF Working Groups {#wg-mod}
 
-The management and moderation of participation channels associated
-with various IETF working groups, including their mailing lists, chat
-channels and in-person, remote, or interim meetings remains primarily a task of
-the relevant group's management, such as WG chairs.
-Group participants may and chairs
-must alert the moderators to problematic behavior that rises to the
-level that some action is needed.
-
-In the interest of timely and consistent moderation,
-moderators may take actions against
-someone in a working group setting, but they must immediately inform
-management of relevant groups, including WG chairs and area directors,
-when they do so.
-
-If working group management and moderators disagree about
-a group moderation action taken by one party unilaterally,
-the matter shall be taken up with the responsible area
-director(s) for resolution, and, when more than one area is involved,
-with the IESG. Based on the result of this consultation, the moderation
-action may be upheld, modified or reverted.
-
-It is anticipated that such disagreements will be exceedingly rare.
-The moderators should respect the views of the group management
-in such cases, and the group management should respect the moderators'
-task of upholding an overall IETF-wide uniformity for
-moderation.
+Primary responsibility of moderation of working group participation
+channels resides with working group management.  Working group
+participants should first report problems to chairs, and absent a
+timely, satisfactory response, to the moderation team (tbd@ietf.org).
```

Comment:
```suggestion
participants should report problems to both the chairs and
to the moderation team (tbd@ietf.org).
```
If issues are only reported to the chairs, how will the mod team learn about them and start the clock to determine if they need to nudge the chairs or act themselves?

Reply:
give me just a day or so.

Reply:
please review.